source/codegen/metadata/nixnetsocket/enums.py

```diff
@@ -1,18 +1,144 @@
 enums = {
-  'OptName': {
+  'OperationalStatus': {
+      'enum-value-prefix': 'OPERATIONAL_STATUS',
+        'values': [
+            {
+                'name': 'DOWN',
+                'value': 0
+            },
+            {
+                'name': 'UP',
+                'value': 1
+            }
+        ]
+    },
+     'IPStackInffoStringFormat': {
+        'enum-value-prefix': 'IPSTACK_INFO_STR_FORMAT',
+        'values': [
+            {
+                'name': 'JSON',
+                'value': 0
+            },
+            {
+                'name': 'TEXT',
+                'value': 1
+            }
+        ]
+    },
+    'AddressFamilies': {
+        'values': [
+            {
+                'name': 'AF_UNSPEC',
+                'value': 0
+            },
+            {
+                'name': 'AF_INET',
+                'value': 2
+            },
+            {
+                'name': 'AF_INET6',
+                'value': 10
+            }
+        ]
+    },
+    'ProtocolFamilies': {
+        'values': [
+            {
+                'name': 'PF_UNSPEC',
```

Comment:
I wonder if it's better for the '..._Unspec' enum values where it's 0 if it's better to just leave it off and have the codegen generate the UNSPECIFIED 0 enum value.

Reply:
I like that idea. 

`UNSPECIFIED` is a widely accepted standard/practice. We'll stick with weird driver names pretty far, but when it's this close, I prefer UNSPECIFIED.